### DISCUSSION TITLE
Cannot change role for first admin

### DISCUSSION THREAD

**Discussion by sebastianlau** [open-webui/open-webui#6128] [General]
# Bug Report

## Installation Method

Docker

## Environment

- **Open WebUI Version:** v0.3.32

- **Operating System:** Ubuntu 20.04.03

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Role switches between Pending/User/Admin

## Actual Behavior:

Role remains the same, error "The requested action has been restricted as a security measure." is presented.

## Description

**Bug Summary:**
The role of the first [admin] account on the system cannot be changed (as an aside that means it can also not be deleted); regardless of whether another admin account exists.

## Reproduction Details

**Steps to Reproduce:**
1. If new instance, register first admin user and log in; else, just log in as first admin user
2. Create another admin user
3. Log in as second admin user
4. Attempt to change first admin users role

## Notes
I'd expect this behaviour to be valid only if a single admin user was present

**Comment by rkeopuhiwa** [comment]
Wanted to check on this and see if its possible to change the primary/first admin in a Open WebUI instance? This is needed as we need to remove the primary/first admin from our instance.

Any help/guidance is appreciated, please let us know.

**Comment by Mte90** [comment]
it is not possible to change the password for that user I get the same error

**Comment by EdiWang** [comment]
Now the error message is: Could not verify primary admin status

**Comment by nathanRamaNoodles** [comment]
1. Use db browser or sqlite3 and edit your `webui.db`'s users table.  For the first user, set the role to `user` and set the `created_at`, `updated_at` to one millisecond greater than the second user.  
2. Restart open-webui.
3. Delete user as normal 😄

**Comment by christopherquenneville** [comment]
PostgreSQL commands for if you're not doing SQLite:

First list admins by creation date
SELECT id, name, email, role, created_at
FROM "user"
WHERE role = 'admin'
ORDER BY created_at;

Second update the admin you want to set as primary
UPDATE "user"
SET created_at = 1742349200
WHERE email = 'email@domain.tld';
